cmake/modules/AddSwift.cmake

```diff
@@ -359,7 +359,7 @@ function(_add_variant_link_flags)
     endif()
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
-        "-ldl" "-llog" "-latomic"
+        "-ldl" "-llog" "-latomic" "-licudata" "-licui18n" "-licuuc"
```

Comment:
I’m not sure linking with those libraries would be the problem. I’ve followed the instructions https://github.com/apple/swift/blob/master/docs/Android.md to run a command line program and though I had to add a “-Xlinker -pie” to the compile command for hello.swift I can get it to work (with a few warnings):

$ adb shell LD_LIBRARY_PATH=/data/local/tmp /data/local/tmp/hello
WARNING: linker: /data/local/tmp/hello: unused DT entry: type 0x6ffffffe arg 0xcb8
WARNING: linker: /data/local/tmp/hello: unused DT entry: type 0x6fffffff arg 0x1
WARNING: linker: libswiftCore.so: unused DT entry: type 0x6ffffffe arg 0xa55c8
WARNING: linker: libswiftCore.so: unused DT entry: type 0x6fffffff arg 0x3
WARNING: linker: libswiftSwiftOnoneSupport.so: unused DT entry: type 0x6ffffffe arg 0x2074c
WARNING: linker: libswiftSwiftOnoneSupport.so: unused DT entry: type 0x6fffffff arg 0x1
WARNING: linker: libicui18n.so: unused DT entry: type 0x6ffffffe arg 0x85e84
WARNING: linker: libicui18n.so: unused DT entry: type 0x6fffffff arg 0x2
WARNING: linker: libicuuc.so: unused DT entry: type 0x6ffffffe arg 0x31c10
WARNING: linker: libicuuc.so: unused DT entry: type 0x6fffffff arg 0x3
WARNING: linker: libc++_shared.so: unused DT entry: type 0x6ffffffe arg 0x2adb0
WARNING: linker: libc++_shared.so: unused DT entry: type 0x6fffffff arg 0x3
Hello, Android

Normally I run an app using gradle, for example the project: https://github.com/SwiftJava/swift-android-samples and it’s associated plugin https://github.com/SwiftJava/swift-android-gradle and things seem to be working with the extra libraries.